src/pages/configurePatch/ConfigurePatchCore.tsx

```diff
@@ -172,7 +172,7 @@ export const ConfigurePatchCore: React.FC<Props> = ({ patch }) => {
         description: patch.description,
         buildVariants: [variants[0]?.name],
         params: patch.parameters,
-        variantTasks: convertPatchVariantTasksToStateShape(variants),
+        variantTasks: initializeTaskState(variants, patch.variantsTasks),
```

Comment:
This assignment is executed when the user clicks "Schedule". If we didn't have [this redirect](https://github.com/evergreen-ci/spruce/blob/main/src/pages/configurePatch/ConfigurePatchCore.tsx#L139) after successful mutation, I think we will actually see the users selected options joined with the contents of variantsTasks. 
I don't think there is a bug here, but it could be helpful to adjust the useEffect so it doesn't run upon a successful mutation.

Reply:
Yeah i can see that being executed because patch is updated. But i think as long as we redirect the user we don't have to worry about it.